packages/flutter/lib/src/material/date_picker_theme.dart

```diff
@@ -73,23 +73,25 @@ class DatePickerThemeData with Diagnosticable {
 
   /// The elevation of a date picker dialog.
   ///
-  /// {@macro flutter.material.material.elevation}
+  /// See also:
+  ///   [Material.elevation], which explains how elevation is related to a component's shadow.
   final double? elevation;
 
   /// The color of the shadow painted below a date picker dialog.
   ///
-  /// {@macro flutter.material.material.shadowColor}
+  /// See also:
+  ///   [Material.shadowColor], which explains how the shadow is rendered.
   final Color? shadowColor;
 
   /// The color of the surface tint overlay applied to the [backgroundColor]
   /// to indicate elevation.
   ///
-  /// {@macro flutter.material.material.surfaceTintColor}
+  /// See also:
+  ///   [Material.surfaceTintColor], which explains how this color is related to
+  ///   [elevation].
   final Color? surfaceTintColor;
 
   /// The overall shape of a date picker dialog.
```

Comment:
Should this have a "See also" for the Material `shape` property?

Reply:
That might be a little confusing, since https://api.flutter.dev/flutter/material/Material-class.html and https://api.flutter.dev/flutter/material/Material/shape.html tie the Material shape property to the Material `type` and `borderRadius`.  Will add a statement that explains that the shadow depends on shape and elevation.